app/Http/Controllers/EmployerController.php

```diff
@@ -12,7 +13,22 @@ class EmployerController extends Controller
      */
     public function index()
     {
-        //
+        $users = DB::table('employers')->join('users', 'users.id', '=', 'employers.user_id');
```

Comment:
Looks very long for me.
Could you extract the search part to private method also make your code based on try catch look at any controller copy and paste the code